angel.py

```diff
@@ -171,27 +174,20 @@ def write_to_csv(index, name01, *player_lists):
             cur_time = time.strftime("%Y-%m-%d %H-%M-%S")
             with open(f"{index} - {name01} - {cur_time}.csv", 'w', newline='') as f: # In Python 3, if do not put newline='' AND choose 'w' instead of 'wb', you will have an empty 2nd row in output .csv file.
                 writer = csv.writer(f, delimiter=',')
-                header = ['Telegram Username','Name','GenderPref','Gender','Interests','2truths1lie','Intro','House','CG','Year','Faculty'] # add header to output csv file
+
+                config_file = open("config.yml", 'r')
```

Comment:
This code is duplicated at 79/80. Could we abstract it, then memoize so the file is only read once